nextflow.config

```diff
@@ -215,6 +213,18 @@ profiles {
         executor.name           = 'local'
         executor.cpus           = 4
         executor.memory         = 8.GB
+        process {
+            resourceLimits = [
+                memory: 8.GB,
+                cpus  : 4,
+                time  : 1.h
+            ]
+        }
+    }
+    gpu {
```

Comment:
are you also running gpu based tests?

Reply:
Wasn't possible until very recently, but could be added I assume 👍🏻

Reply:
Let's open a ticket for this.